en/docs/administer/changing-to-mariadb.md

```diff
@@ -1,85 +1,164 @@
 # Changing to MariaDB
 
-The following sections describe how to replace the default H2 database
-with MariaDB, which is a drop-in replacement for MySQL.
-
-!!! tip "Before you begin"
-    You need to set up MariaDB before following the steps to configure your
-    product with it. For more information, see [Setting up
-    MariaDB](../../administer/setting-up-mariadb).
+By default, WSO2 Identity Server uses the embedded H2 database as the database
+for storing user management and registry data. Given below are the steps
+you need to follow in order to use MariaDB for this purpose.
     
-### Setting up datasource configurations
-
-A datasource is used to establish the connection to a database. By
-default, `         WSO2_CARBON_DB        ` datasource is used to connect
-to the default  H2 database, which stores registry and user management
-data. After setting up the MariaDB database to replace the default H2
-database, either change the default configurations of the
-`          WSO2_CARBON_DB         ` datasource, or configure a new datasource to point it to the new database as explained below.
-
-#### Changing the default datasource
 
-Follow the steps below to change the type of the default datasource.
+## Setting up datasource configurations
+
+A datasource is used to establish the connection to a database. By
+default, `WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
+to the default  H2 database. 
+
+- `WSO2_SHARED_DB` - The database which stores registry and user management
+                     data.
+- `WSO2_IDENTITY_DB` - The database specific for the identity server which stores
+                       identity related data
+                       
+After setting up MariaDB database. You can point the `WSO2_IDENTITY_DB` or 
+`WSO2_SHARED_DB` or both to that MariaDB database by following below instructions.
+
+!!! note
+       
+    WSO2 Identity Server currently does not provide specific MariaDB SQL scripts.However
```

Comment:
does not provide scripts specific to MariaDB SQL.

Reply:
done